### DISCUSSION TITLE
i have problem when run httpx ports

### DISCUSSION THREAD

**Discussion by shadihh9** [projectdiscovery/httpx#333] [General] [answered]
when run echo "emaple" | metabigor net --org -o emaple_result.txt ; mapcidr -l emaple_result.txt -o emaple_output.txt ; httpx -l  emaple_output.txt -ports 1090,1098,1099,4444,11099,47001,47002,10999,7000-7004,8000-8003,9000-9003,9503,7070,7071,45000,45001,8686,9012,50500,4848,11111,4444,4445,4786,5555,5556,8880,8088,6379,2375,8983,8383,4990,9000,8500,6066 -threads 9000  | anew emaple_ips | nuclei -c 500 -l emaple_ips -t /root/nuclei-templates/ -etags openssh -severity critical,high -o emaple_ips_results.txt
dont work httpx

**Comment by ehsandeep** [comment, marked as answer]
@shadihh9 it's unclear to say what's going on as you running multiple tools and the information doesn't contain any information related to the error.

apart from that **httpx** does have a port scanning feature for convincing use but is not a primary tool to run **port scanning** at mass scale, instead you should put [naabu](https://github.com/projectdiscovery/naabu) in between which will also improve the scan time of your workflow.

```
 echo "emaple" | metabigor net --org -o emaple_result.txt ; mapcidr -l emaple_result.txt -o emaple_output.txt ; naabu -iL emaple_output.txt -p 1090,1098,1099,4444,11099,47001,47002,10999,7000-7004,8000-8003,9000-9003,9503,7070,7071,45000,45001,8686,9012,50500,4848,11111,4444,4445,4786,5555,5556,8880,8088,6379,2375,8983,8383,4990,9000,8500,6066 -o naabu_output.txt ; httpx -l naabu_output.txt -threads 200 | anew emaple_ips | nuclei -c 500 -l emaple_ips -t /root/nuclei-templates/ -etags openssh -severity critical,high -o emaple_ips_results.txt
```
